sec1_intro_and_setup/test_app/config/environments/test.rb

```diff
@@ -0,0 +1,59 @@
+require "active_support/core_ext/integer/time"
```

Comment:
Whats this file for?

Reply:
Your answer doesn't explain the usage of this file.

Reply:
This file lets you configure how your application will work in testing environments.